core/src/main/java/org/springframework/security/authentication/DefaultAuthenticationEventPublisher.java

```diff
@@ -160,6 +160,25 @@ public void setAdditionalExceptionMappings(Properties additionalExceptionMapping
 		}
 	}
 
+	/**
+	 * Overload of {@link #setAdditionalExceptionMappings(Properties)}}, providing the ability
+	 * to map authentication exceptions and failure authentication events using a {@link Map}
+	 *
+	 * @param mappings where keys are exception classes and values are event classes.
```

Comment:
Will you please add a `@since 5.3` attribute here? The [contribution guidelines](https://github.com/spring-projects/spring-security/blob/master/CONTRIBUTING.md#use-since-tags-for-newly-added-public-api-types-and-methods) show where to place this.